src/Sylius/Bundle/ApiBundle/test/config/packages/_sylius.yaml

```diff
@@ -0,0 +1,94 @@
+imports:
```

Comment:
Should this file be called `_sylius`? I can see all kinds of configs here despite sylius one. Let's rename this file to `config.yaml`. Also, can you put config of `api_platform.yaml` and `security.yaml` here as well?